src/listing-image-slider.html

```diff
@@ -18,23 +18,18 @@
     <body>
 
         <main class="container">
-          <section class="img-slider">
+          <section class="img-slider" ng-app="slider-module" ng-controller="post-image">
             <div class="img-main">
               <a href="#" id="prev"><div class = "arrow-case"></div><i class="fa fa-arrow-left"></i></a>
               <img class= "img-active" src="https://img0.etsystatic.com/047/0/6292388/il_570xN.736798340_td30.jpg">
               <a href="#" id="next"><div class = "arrow-case"></div><i class="fa fa-arrow-right"></i></a>
             </div>
-            <ul class="img-list">
-              <li class="img-list-items"><img class="active" src="http://lorempixel.com/32/32/animals"></li>
-              <li class="img-list-items"><img src="http://lorempixel.com/32/32/abstract"></li>
-              <li class="img-list-items"><img src="http://lorempixel.com/32/32/people"></li>
-              <li class="img-list-items"><img src="http://lorempixel.com/32/32/nature"></li>
-              <li class="img-list-items"><img src="http://lorempixel.com/32/32/animals"></li>
-              <li class="img-list-items"><img src="http://lorempixel.com/32/32/people"></li>
-              <li class="img-list-items"><img src="http://lorempixel.com/32/32/abstract"></li>
-              <li class="img-list-items"><img src="http://lorempixel.com/32/32/people"></li>
-              <li class="img-list-items"><img src="http://lorempixel.com/32/32/animals"></li>
-              <li class="img-list-items"><img src="http://lorempixel.com/32/32/abstract"></li>
+            <ul class="img-list" ng-repeat= "img in item">
```

Comment:
this is working actually, for some reason it repeats the `<li>` too much than i need, im thinking of replacing it with other "ng-" then it may actually work properly.